api-generation/src/main/scala/ai/h2o/sparkling/api/generation/python/MOJOModelFactoryTemplate.scala

```diff
@@ -74,31 +75,42 @@ object MOJOModelFactoryTemplate extends ((Seq[AlgorithmSubstitutionContext]) =>
     }
 
     val models = s"""
-       |class H2OMOJOModel(H2OMOJOModelParams, H2OMOJOModelBase, H2OMOJOModelFactory):
+       |class WithCVModels(H2OMOJOModelFactory):
```

Comment:
@michalkurka Yes, CV models are saved always when cross-validation is applied. Np, I can add a flag parameter saying whether the CV models will be appended to the main model or not. Could you share more details why it should be optional? impact on performance? The size of the final model?

Reply:
You are typically primarily interested in metrics of the CV fold models, you might also want to preserve the holdout  predictions (in case you want to stack the models), the actual model might be useful in specific cases but I don't believe this is common enough to warrant an n-times increase of the size of the saved data.

It would make sense to save the metrics always but not the actual models.

Reply:
Thanks for  the clarification! I've add a parameter to both PRs as requested.